components/availability/src/Availability.svelte

```diff
@@ -338,56 +485,101 @@
+        border: 1px solid rgba(0, 0, 0, 0.1);
+        border-bottom: none;
 
         .slot {
-          border: 1px solid #fff;
-          background: #eee;
+          border: none;
+          border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+          background: transparent;
           position: relative;
           align-items: center;
           justify-content: center;
           align-content: center;
           font-family: sans-serif;
-          font-size: 0.8rem;
 
           &.selected {
-            background-color: yellow;
+            background-color: purple;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.25);
           }
 
-          &.free {
-            border: 1px solid green;
-          }
-          &.partial {
-            border: 1px solid #ccc;
-          }
           &.busy {
-            border: 1px solid red;
             cursor: not-allowed;
-            opacity: 0.3;
           }
         }
       }
     }
     button.confirm {
       grid-column: -1 / 1;
     }
+
+    &.allow_booking {
+      .slot:not(.busy):hover {
+        box-shadow: 0 0 10px rgba(0, 0, 0, 0.25);
+        cursor: pointer;
+      }
+    }
   }
 </style>
 
 <nylas-error {id} />
-<main bind:this={main} bind:clientHeight class:ticked={show_ticks}>
+<main
+  bind:this={main}
+  bind:clientHeight
+  class:ticked={show_ticks}
+  class:allow_booking
+>
   {#if show_ticks}
     <ul class="ticks">
       {#each ticks as tick}
-        <li class="tick">{tick.toLocaleTimeString()}</li>
+        <li class="tick">
+          {tick.toLocaleTimeString("default", {
+            hour: "numeric",
+            minute: "numeric",
+          })}
+        </li>
       {/each}
     </ul>
   {/if}
   <div class="days">
     {#each days as day}
       <div class="day">
         <header>
-          <h2>{new Date(day.timestamp).toLocaleDateString()}</h2>
+          <h2>
+            <span class="date"
+              >{new Date(day.timestamp).toLocaleString("default", {
+                day: "numeric",
+              })}</span
+            >
+            <span class="weekday"
+              >{new Date(day.timestamp).toLocaleString("default", {
+                weekday: "long",
+              })}</span
+            >
+          </h2>
         </header>
+        <div class="epochs">
+          {#each day.epochs as epoch}
+            <div
+              class="epoch {epoch.status}"
+              style="height: {epoch.height}%; top: {epoch.offset}%;"
+              data-available-calendars={epoch.available_calendars.toString()}
+              data-start-time={new Date(epoch.start_time).toLocaleString()}
+              data-end-time={new Date(epoch.end_time).toLocaleString()}
+            >
+              <div class="inner">
+                <span class="available-calendars">
+                  <span
+                    >{epoch.available_calendars.length} of {allCalendars.length}</span
+                  >
+                  {#each epoch.available_calendars as calendar}<span
```

Comment:
This should be considered a temporary design flourish (hovering "x of y available" to reveal email addresses of available calendars) until we hook into the Contacts API and fetch avatars to show

Reply:
The info for which calendars are available isn't accessible to anyone not using a mouse. Can you add the calendars of available people to the aria-label for the slot?

Reply:
@heathergh added the following to the `epoch` element:

```
aria-label="{new Date(epoch.start_time).toLocaleString()} to {new Date(epoch.end_time).toLocaleString()}; Free calendars: {epoch.available_calendars.toString()}"
```